<?xml version="1.0"?>
<launch>
	<node name="map_matcher" pkg="local_tutorials" type="map_matcher" output="screen" >
		<param name="map_file" value="/home/amsl/ros_catkin_ws/maps/d_kan_indoor.pcd" />
		<param name="voxel_size" value="0.5" />
	</node>
</launch>


